tests/test_playbooks/host_errata_info.yml

```diff
@@ -0,0 +1,77 @@
+---
+- hosts: localhost
```

Comment:
What part the playbook here sets up the host inside Foreman?

Reply:
None. I did that manually. And could document that in a comment?